internal/featuregates/mapper_test.go

```diff
@@ -58,8 +58,8 @@ func TestFormatAsEnabledArgs(t *testing.T) {
 		},
 		{
 			name:     "multiple feature gates",
-			in:       []string{"testGate1", "testGate2", "testGate3"},
-			expected: "testGate1=true,testGate2=true,testGate2=true",
```

Comment:
How did this test pass before? Input was 1,2,3. Expected output was 1,2,2?

Reply:
I don't believe it was run!

Reply:
I added `./internal` into `GO_TEST_PACKAGES` variable in the Makefile

Reply:
And apparently, we don't have a unit-test CI‽‽

Reply:
🙃